src/constants/markets.ts

```diff
@@ -48,20 +48,6 @@ export enum MarketFilters {
   NEW = 'new',
   PREDICTION_MARKET = 'prediction-market',
 
-  // Existing Custom Tags from public/configs/markets.json
-  // TODO: Remove when metadataService is default
-  PREDICTION_MARKET_DEPRECATED = 'Prediction Market',
```

Comment:
is this object stored in persisted state anywhere? If so, need to migrate to remove them

Reply:
nope. `marketFilter` is used in `state/perpetual` which is not included in redux-persist